storage_controller/migrations/2024-12-12-212515_safekeepers_scheduling_policy/up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE safekeepers ADD scheduling_policy VARCHAR NOT NULL DEFAULT 'disabled';
```

Comment:
Can we drop the `active` column too?

Reply:
I would like to drop it, but I saw this warning in the docs: https://docs.neon.build/storage/internals_controller.html?#diesel-tip-migrations . I don't know diesel well enough to do a safe transition to the new state.

Reply:
I wrote that warning :smile:

Let's apply it to this scenario. We remove the `active` column and restart the storage controller.
1. New storage controller starts-up
2. Old storcon receives step-down request
3. New db migrations run
4. New storcon marks itself as the leader

We should look for situations where, after the migrations have been applied, the old code would query the
`active` column. For example, if there's a network partition at step (2) and a `list_safekeepers` request lands on the old node. Does anything call `list_safekeepers` right now (don't think so)? If that's the case we should be fine.

Reply:
Indeed. We can disable the new safekeeper GH action while releasing.

Reply:
could we do this in a followup? I think it's better to not keep the column number constant.

Reply:
Don't think it makes a difference, but sure.